zavod/docs/best_practices/xpath_and_html.md

```diff
@@ -0,0 +1,30 @@
+# XPath and HTML
+
+## Selector Design
+Use semantic selectors that are resilient to HTML changes:
```

Comment:
```suggestion
Try to make the selectors specific enough to reliably select the correct content (to avoid e.g. the wrong thing being stuffed in a person's `name` property) while making them as semantic, concise, and general as possible to be maintainable and resillient against irrelevant changes in the HTML.
```